### PR TITLE
Add backend changes with search funtionality

### LINKED ISSUE #28: Add search functionality

### PR DESCRIPTION
**Pull request by AsithaIndrajith**
Closes #23  Closes #15 Closes #28

### CODE REVIEW

File: com-dict-client/src/components/WordHome/word.js [hunk 1]
```diff
@@ -1,157 +1,151 @@
...
 
-function Word()
-{
-    return(
-
-        <div>
+function Word() {
+  return (
+    <Row>
+<<<<<<< HEAD
```
Comment (shehand) [inline com-dict-client/src/components/WordHome/word.js:21]: There is a commit conflict here.

File: com-dict-client/src/components/WordHome/word.js [hunk 2]
```diff
@@ -1,157 +1,151 @@
...
+              <Text>English</Text>
+=======
+          <Col xl={12} lg={12} style={{ textAlign: "right" }}>
+            <SocialShare />
+          </Col>
+        </Row>
+        
+        <Row align="middle">
+              <Text >English</Text>
+>>>>>>> e625349a589bde16508b9448e56021c08cd0493f
```
Comment (shehand) [inline com-dict-client/src/components/WordHome/word.js:61]: Commit conflicts end at here for this file

File: com-dict-client/src/components/WordSearch/Word.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+// } from "./node_modules/@ant-design/icons";
+import SelectLang from "../Search/SelectLang";
+import SocialShare from "./SocialShare";
+
+const { Title, Text } = Typography;
+
+function Word() {
+  return (
+    <Row>
+<<<<<<< HEAD
```
Comment (shehand) [inline com-dict-client/src/components/WordSearch/Word.js:21]: There are commit conflicts in this file also

File: com-dict-client/src/components/logo/info.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
+Hope you enjoy your new logo, here are the people that
```
Comment (shehand) [inline com-dict-client/src/components/logo/info.txt:2]: I'm not sure whether we should add this file or not. @Wathsara?

**Comment by AsithaIndrajith** [comment]
Merged conflicts and remove info.txt @shehand @Wathsara

**Comment by shehand** [comment]
@Wathsara can you double-check changes? I might have missed some files.

**Pull request merged**